### DISCUSSION TITLE
prism-nord.css is missing for plugin-prism ?

### DISCUSSION THREAD

**Discussion by sytolk** [Milkdown/milkdown#1107] [Q&A]
I have use "@milkdown/plugin-prism": "7.2.4" and expect that CodeBlock will be highlight like in playground but prism-nord.css is not loaded from my side. I have searching in website https://github.com/Milkdown/website/blob/main/docs/api-src/plugin-prism.md#usage
but can't fint where is prism-nord.css loaded ?

**Comment by Saul-Mirone** [comment]
The prism theme is imported from cdn https://github.com/Milkdown/website/blob/main/src/pages/_document.tsx
